libs/sdk-bindings/bindings-react-native/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "bindings-react-native"
+version = "0.0.10"
```

Comment:
Isn't the correct version `0.0.13`?

https://github.com/breez/breez-sdk-rn-generator/blob/59b4ba2a92bca5c2cb83f42d44f8a28583bd6566/Cargo.toml#L3C12-L3C18

Reply:
True, fixed